services/search/pkg/query/bleve/compiler.go

```diff
@@ -12,18 +12,22 @@ import (
 )
 
 var _fields = map[string]string{
-	"rootid":   "RootID",
-	"path":     "Path",
-	"id":       "ID",
-	"name":     "Name",
-	"size":     "Size",
-	"mtime":    "Mtime",
-	"mimetype": "MimeType",
-	"type":     "Type",
-	"tag":      "Tags",
-	"tags":     "Tags",
-	"content":  "Content",
-	"hidden":   "Hidden",
+	"rootid":      "RootID",
+	"path":        "Path",
+	"id":          "ID",
+	"name":        "Name",
+	"size":        "Size",
+	"mtime":       "Mtime",
+	"mimetype":    "MimeType",
+	"type":        "Type",
+	"tag":         "Tags",
+	"tags":        "Tags",
+	"content":     "Content",
+	"hidden":      "Hidden",
+	"album":       "Audio.album",
+	"albumArtist": "Audio.albumArtist",
+	"artist":      "Audio.artist",
+	"title":       "Audio.title",
```

Comment:
`.` seems to have a specific meaning in kql, so e.g. `audio.artist` cannot be used?

Reply:
what exactly are you trying to search for?

`GET /drives/{drive-id}/root/search(q='{search-text}')` or `GET /me/drive/search(q='Contoso Project')` to include all drives the user has access to is a full text search that includes content.

I think `GET /drives/{drive-id}/items?$filter=audio/artist eq Motörhead` better captures what you are trying to do ...

Reply:
Well, I want to search for "artist:Motörhead" and get all songs that have Motörhead as artist and none that only have "(Motörhead Cover)" in the title